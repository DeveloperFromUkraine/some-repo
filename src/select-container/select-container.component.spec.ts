import { ComponentFixture, TestBed } from '@angular/core/testing';
import { SelectContainerComponent } from '../index';
import { ComponentTest } from '../../test/test-bed/component';

describe('Select Container', () => {
  let fixture: ComponentFixture<SelectContainerComponent>;

  beforeEach(async () => {
    await ComponentTest.createTestBed([], [SelectContainerComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(SelectContainerComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
