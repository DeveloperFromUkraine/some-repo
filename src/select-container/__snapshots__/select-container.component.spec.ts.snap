// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectContainerComponent should match snapshot 1`] = `
<ign-select-container>
  
</ign-select-container>
`;
